pointcloud/octree/basic.go

```diff
@@ -0,0 +1,171 @@
+package octree
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	"github.com/pkg/errors"
+
+	pc "go.viam.com/rdk/pointcloud"
+)
+
+// octree data is the structure of an octree.
```

Comment:
[q] What are you referring to here with "octree data"?

The reason why I'm asking is that this comment is confusing and I think it could be more descriptive (in ways I can't explain until I know the answer to my question)

Reply:
will make more descriptive